plugins/catalog-backend/src/database/migrations/20200511113813_init.ts

```diff
@@ -48,6 +48,7 @@ export async function up(knex: Knex): Promise<any> {
           .references('id')
           .inTable('locations')
           .nullable()
+          .onDelete('CASCADE')
```

Comment:
This is not what we want to do though! You should set the relation to null instead, we do not want to accidentally delete entire trees of important things.

Reply:
Got it